pkg/template/processor.go

```diff
@@ -0,0 +1,193 @@
+package template
+
+import (
+	"context"
+	"fmt"
+	"gopkg.in/yaml.v2"
```

Comment:
we should probably reorganize the imports here (but that's something minor)

Reply:
imports, imports, bloody imports :hankey: :gun: 
could you please remind me what is the expected format again? I always forgot that as I only use "the format option" in IDE

Reply:
it should be something like this:
```
import (
    <sdtlib package>
    <sdtlib package>
    ...
    <blank line>
    <our package>
    <our package>
    ...
    <third party package>
    <third party package>
    ...
)
```

Reply:
does it look better? 70877f8